### PR TITLE
[Aptos Node] Bump aptos-node version.

### PR DESCRIPTION
**Pull request by JoshLind**
### Description
This PR bumps the `aptos-node` crate version to `1.7` given that this is the latest release.

### Test Plan
Existing test infrastructure.

### CODE REVIEW

File: aptos-node/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "aptos-node"
 description = "Aptos node"
-version = "1.6.0"
+version = "1.7.0"
```
Comment (ibalajiarun) [inline aptos-node/Cargo.toml:4]: We don't bump patch version?

Comment (JoshLind) [reply aptos-node/Cargo.toml:4]: No -- we've typically only done major release versions. But if we can automate and/or test this, minor versions make sense to me 😄

**Comment by brianolson** [comment]
I wish there was a build script or test that ensured the version agreed in
```
aptos-move/aptos-release-builder/data/release.yaml
aptos-node/Cargo.toml
```

**Comment by JoshLind** [comment]
> I wish there was a build script or test that ensured the version agreed in

Yeah, I think someone is going to need to tackle this in a more holistic way (in the future). But, I won't be doing that in this PR 😄

**Review by brianolson — APPROVED** [review]

**Pull request merged**